metrics/ci_integrations/lib/integration/validation/model/config_field_target_validation_result.dart

```diff
@@ -0,0 +1,117 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:ci_integration/integration/validation/model/config_field_validation_conclusion.dart';
+import 'package:meta/meta.dart';
+import 'package:metrics_core/metrics_core.dart';
+
+/// A class that represents a [TargetValidationResult] for CI integrations
+/// config field.
+class ConfigFieldTargetValidationResult<T> extends TargetValidationResult<T> {
+  /// A flag that indicates whether this is a successful target validation
+  /// result.
+  bool get isSuccess => conclusion == ConfigFieldValidationConclusion.valid;
```

Comment:
🔖 
```suggestion
  bool get isValid => conclusion == ConfigFieldValidationConclusion.valid;
```
Does it make sense to match the constructor names?